doc/providers/form.rst

```diff
@@ -53,12 +53,12 @@ Registering
         composer require symfony/form
 
     If you are going to use the validation extension with forms, you must also
-    add a dependency to the ``symfony/config`` and ``symfony/translation``
```

Comment:
Are you sure that translation is not a requirement? Or is it a transitive requirement of validator?